spooky-oysters/bangazon-web-app
# Kb products form with local delivery - READY FOR REVIEW

## Link to Ticket
#1 and #3 

## Description of Proposed Changes

- User logs in and then will see "sell a product"
- User enters product details
- Product is added to database

## Steps to Test

Outline the steps to test

```sh
git fetch --all
git checkout kb-localDelivery
```
1. Build
1. Click "sell a product"
1. Enter info
1. Check database to make sure it was added correctly `select * from product`

Still TODO:
1. Redirect to product detail after submitting new product
2. Validation on form fields

## Impacted Areas in Application

List general components of the application that this PR will affect:

* ProductsController.cs
* Create.cshtml
* ProductViewModel.cs

## Mentions @username

Tag users that need to review this code

## Definition of Done

1. The project must be fully documented. This includes the following:
    1. Complete README that documents the steps to install the code, how to install any dependencies, or system configuration needed.
    1. Every class must be documented with purpose, author, and methods.
    1. Every method must be documented with purpose and argument list - which itself must contain a short purpose for each argument.
1. The project must be able to run fully, and without errors, on each teammate's system.
1. Fulfills every requirement.
Every line of code has been peer reviewed.
1. For projects that require unit testing, core functionality must be identified and have at least one test for each.
